dagjose/dagjose_test.go

```diff
@@ -71,7 +71,7 @@ func validJWSGen() *rapid.Generator {
 		}
 		gojoseJws, err := signer.Sign(link.Bytes())
 		if err != nil {
-			panic(fmt.Errorf("Error signing ValidJWS: %v", err))
+			panic(fmt.Errorf("error signing ValidJWS: %v", err))
```

Comment:
we should remove these panics! https://github.com/golang/go/wiki/CodeReviewComments#dont-panic

let's have the method return `(*rapid.Generator, error)` and handle the error at the call site.

Reply:
Given that the call site is in a test function, is this worth it? Tests are designed to panic in the case of failure anyway so I'm not sure what the error handling buys you.

Reply:
yes it's worth it, there are idiomatic methods to handle test failures (`t.Fail(..)`, `require.NoError()`)

Reply:
I agree, I opened this https://github.com/ceramicnetwork/go-dag-jose/pull/14